community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -689,10 +689,31 @@ public IdType apply( String from )
         @Override
         public Character apply( String value ) throws RuntimeException
         {
-            if ( value.equals( "TAB" ) )
+            // Parse "raw" ASCII character style characters:
+            // - \123 --> character with id 123
+            // - \t   --> tab character
+            if ( value.startsWith( "\"" ) && value.length() > 0 )
```

Comment:
Also, do raw values really start with a `"`? I would have thought a `\`? So shouldn't it be `value.startsWith( "\\" )` ?